lib/sales_analyst.rb

```diff
@@ -1,13 +1,13 @@
 class SalesAnalyst
-  def initialize(merchant_repo, item_repo, invoice_repo)
-    @merchant_repo = merchant_repo
-    @item_repo = item_repo
-    @invoice_repo = invoice_repo
+  def initialize(engine)
```

Comment:
Now I understand what you mentioned to us in Slack about passing the Engine instead of the repos!